pkg/app/signalingserver.go

```diff
@@ -32,11 +32,15 @@ import (
 // communicating with the host orchestrator in the appropriate instance and
 // forwarding all requests to it.
 type ForwardingSignalingServer struct {
+	interceptPath   string
```

Comment:
nit: `interceptedFilesPath` is more clear

Reply:
used `connectorStaticFilesPath` as discussed offline.